test/dsl/dsl_options.jl

```diff
@@ -436,7 +436,8 @@ let
     end
 
     @parameters k k2
-    @variables s x D(x) E(s) F(s,x)
+    @parameters s x
```

Comment:
Generally I did not merge the new independent-variable-paraemter declarations with old parameter declarations. It seems desirable to create a `@independent_variables` declaration or similar for these in the future, so better keep them separate for when that happens.

Reply:
Sounds like that macro will happen, so we can update for it once available. For now we just want to ensure we don't get test failures or such from the change (and even if they change it to just be a warning now it would be good to avoid warnings in the test logs).